crates/biome_service/src/configuration/overrides.rs

```diff
@@ -215,6 +215,15 @@ pub struct OverrideFormatterConfiguration {
     #[bpaf(long("indent-width"), argument("NUMBER"), optional)]
     pub indent_width: Option<u8>,
 
+    /// The type of line ending.
+    #[serde(skip_serializing_if = "Option::is_none")]
+    #[bpaf(
+        long("line-ending"),
+        argument("lf|line-feed|crlf|carriage-return-line-feed|cr|carriage-return"),
```

Comment:
I have mixed feelings about supporting two values for the same option e.g. `lf` and `line-feed` for `LineEnding::LineFeed`.  I think we should use only one value.

Personally, I would use the more verbose options (I like verbosity), but the others are fine too. I don't have strong opinions, whatever you choose I am fine with it.

Reply:
i only added the long-style options to match the already-defined enums, then later tried `lf` and similar as shortcuts, i don't like the long-style names either, i'm gonna try removing them